problems/objects.js

```diff
@@ -187,15 +268,55 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = (str) => {
+  let occurence = {};
+  let newArr = str.split(" ")
+  let newStr = newArr.join("")
+  for (let i = 0; i < newStr.length; i++) {
+    const el = newStr[i].toLowerCase();
+   if (occurence[el] !== undefined) {
+      occurence[el] += 1;
+    } else {
+      occurence[el] = 1;
+    }
+  }
+    return occurence;
+  }
+
+
 
 /**
  * Takes in an array and returns the most common element.
  * @param {Array} array - array of elements
  * @returns {(number|string)} Most common element
  */
 
-const mostCommonElement = () => {};
+const mostCommonElement = (array) => {
+
+let obj = {}
+for(let i = 0; i < array.length; i++) {
+  let el = array[i]
+if(obj[el] !== undefined) {
+  obj[el] +=1;
+} else {
+  obj[el] = 1
+}
```

Comment:
Make sure certain things are indented where need, you can use a code formatter like prettier, it can be installed into you vs code through the extension tab